NAMESPACE

```diff
@@ -1,10 +1,15 @@
+# Generated by roxygen2: do not edit by hand
+
+export(assertFalse)
 export(assertThat)
 export(assertTrue)
-export(assertFalse)
 export(closeTo)
-export(identicalTo)
+export(deparsesTo)
+export(emitsWarning)
 export(equalTo)
+export(identicalTo)
 export(instanceOf)
-export(isTrue)
 export(isFalse)
+export(isTrue)
```

Comment:
Do we need these? Are `isTRUE()` and `isFALSE()` not sufficient? I think it is too confusing to introduce new functions that differ only in case.

Reply:
I think there's a difference that `isTrue()` and `isFalse()` are matcher functions. `assertThat(TRUE, isTrue())` is a possible test but `assertThat(TRUE, isTRUE())` is not. You explained well in the comments above that "matcher functions ... do not "check" any values, they return functions which check". Also, I thought those matchers are used in Renjin and didn't want to do any change on the API, as informed in the PR text.

Reply:
Strictly speaking, `identicalTo` and `equalTo` a matcher-generating functions. They are only needed when the matcher needs to be parametrized. `isTRUE` and `isFALSE` already match against a value, and don't parameters, so:

```
assertThat(x, isTRUE)
```
works perfectly.

Reply:
That's totally legit for `isTRUE()`, but `isFALSE()` is introduced to R in version 3.5.0. Users using any version older than that has to write a new matcher in order to check values against whether they are false.

Also `isFALSE()` doesn't exist in Renjin yet. https://github.com/bedatadriven/renjin/issues/464